### PR TITLE
Ignore `build` and `node_modules` folder for `webpack --watch`

### PR DESCRIPTION
**Pull request by LukasKalbertodt**
Before, the page was reloaded two additional times, uselessly, as webpack was triggered by changes in `build`.

### CODE REVIEW

**Comment by owi92** [comment]
This still reloads twice on my end.. or wait, is it supposed to do that? Because you say before it reloaded two **additional** times, so three times total?

**Comment by LukasKalbertodt** [comment]
Well, it attempted to reload three times in total, but since two were very close to another, only two reloads were visibile in the browser in total. But with this PR it should be down to one. 
Are you sure you restartet `x.sh`? And what did you change? I tested with changing some random margin or sth like that. In the frontend.

**Comment by owi92** [comment]
Yes I restarted 'x.sh' (also containers for good measure lol). I changed a string.. but let me try again with a margin 🤷

**Comment by owi92** [comment]
Same outcome, still two visible reloads (and three in total).

**Review by owi92 — APPROVED** [review]
Turns out the issue is on my end.
